src/components/RichTextEditor/RichTextEditor.stories.tsx

```diff
@@ -101,7 +109,23 @@ const RichTextEditorTemplate: StoryFn<RichTextEditorProps> = (args: RichTextEdit
 
 const allPlugins = new PluginComposer();
 allPlugins
-    .setPlugin([new SoftBreakPlugin(), new ParagraphPlugin(), new TextStylePlugin()])
+    .setPlugin([
```

Comment:
add this or Paragraph() also in `With List` story (or remove dropdown completely)